src/signal_scope/plotwidget.cpp

```diff
@@ -7,6 +7,7 @@
 #include "signaldescription.h"
 #include "pythonchannelsubscribercollection.h"
 
+#include <float.h>
```

Comment:
see comment about std::numeric_limits.  You could remove this include and replace with:

`#include <limits>`